service-commands/scripts/setup-instance.sh

```diff
@@ -107,8 +107,29 @@ case "$service" in
 
 		read -p "Configure local /etc/hosts? [y/N] " -n 1 -r && echo
 		if [[ "$REPLY" =~ ^[Yy]$ ]]; then
-			sudo -E AUDIUS_REMOTE_DEV_HOST=$(get_ip_addr $provider $name) node $PROTOCOL_DIR/service-commands/scripts/hosts.js
-			exit 1
+			IP=$(get_ip_addr $provider $name)
```

Comment:
nit, but might be good to factor these into named bash functions - `configure_etc_hosts`, `set_ssh_serveralive`, `setup_zsh`